src/test/groovy/net/fabricmc/loom/BuildUtils.groovy

```diff
@@ -24,7 +24,7 @@ minecraft {
 dependencies {
 	//to change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:\${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:\${project.yarn_mappings}:v2"
```

Comment:
why no classifier again?

Reply:
It's now part of ${project.yarn_mappings} so you can change version, look at the next change